src/Servers/IIS/IIS/test/Common.FunctionalTests/HttpsTests.cs

```diff
@@ -106,12 +106,32 @@ public async Task HttpsPortCanBeOverriden()
                         .SetAttributeValue("bindingInformation", $":{TestPortHelper.GetNextSSLPort()}:localhost");
                 });
 
-            deploymentParameters.WebConfigBasedEnvironmentVariables["ASPNETCORE_HTTPS_PORT"] = "123";
+            deploymentParameters.WebConfigBasedEnvironmentVariables["ASPNETCORE_ANCM_HTTPS_PORT"] = "123";
 
             var deploymentResult = await DeployAsync(deploymentParameters);
             var client = CreateNonValidatingClient(deploymentResult);
 
-            Assert.Equal("123", await client.GetStringAsync("/HTTPS_PORT"));
+            Assert.Equal("123", await client.GetStringAsync("/ANCM_HTTPS_PORT"));
+        }
+
+        [ConditionalFact]
+        [RequiresNewShim]
+        public async Task HttpsPortOldValueDoesNotRedirect()
+        {
+            var deploymentParameters = Fixture.GetBaseDeploymentParameters(HostingModel.OutOfProcess);
+
+            deploymentParameters.AddServerConfigAction(
+                element => {
+                    element.Descendants("bindings")
+                        .Single()
+                        .GetOrAdd("binding", "protocol", "https")
+                        .SetAttributeValue("bindingInformation", $":{TestPortHelper.GetNextSSLPort()}:localhost");
+                });
+
+            var deploymentResult = await DeployAsync(deploymentParameters);
+            var client = CreateNonValidatingClient(deploymentResult);
+
+            Assert.Equal("NOVALUE", await client.GetStringAsync("/HTTPS_PORT"));
```

Comment:
Does ANCM_HTTPS_PORT have a value set in this test? Either way, can we verify that the is/isn't set in this test?

Reply:
My assumption is that ANCM_HTTPS_PORT should equal whatever TestPortHelper.GetNextSSLPort() returned, right?

Reply:
Yeah, this was already done by another test. I now consolidated them.